language/move-vm/types/src/values/values_impl.rs

```diff
@@ -148,7 +148,7 @@ pub struct Vector(Container);
 #[derive(Debug)]
 pub struct StructRef(ContainerRef);
 
-/// A generic Move reference that offers two functinalities: read_ref & write_ref.
+/// A generic Move reference that offers two functionalities: read_ref & write_ref.
```

Comment:
Wow... soooo many typos... enough to earn me residency in the clowntown...

Anyway, good catch!